diff -dur -x '*.orig' VulkanTools-sdk-1.0.32.0.orig/CMakeLists.txt VulkanTools-sdk-1.0.32.0/CMakeLists.txt
--- VulkanTools-sdk-1.0.32.0.orig/CMakeLists.txt	2016-11-23 14:19:14.000000000 +0100
+++ VulkanTools-sdk-1.0.32.0/CMakeLists.txt	2016-11-23 14:19:30.000000000 +0100
@@ -186,6 +186,7 @@
         set (SPIRV_TOOLS_SEARCH_PATH "${EXTERNAL_SOURCE_ROOT}/spirv-tools/${BUILDTGT_DIR}/source/${RELEASE_DECORATION}")
         set (SPIRV_TOOLS_DEBUG_SEARCH_PATH "${EXTERNAL_SOURCE_ROOT}/spirv-tools/${BUILDTGT_DIR}/source/${DEBUG_DECORATION}")
     endif()
+    set(DisplayServer Xcb)
 else()
     #non windows
     if(CUSTOM_GLSLANG_BIN_ROOT)
