diff -dur -x '*~' vulkan-sdk-1.0.3.0.orig/VulkanTools/CMakeLists.txt vulkan-sdk-1.0.3.0/VulkanTools/CMakeLists.txt
--- vulkan-sdk-1.0.3.0.orig/VulkanTools/CMakeLists.txt	2016-03-02 19:06:25.059777274 +0100
+++ vulkan-sdk-1.0.3.0/VulkanTools/CMakeLists.txt	2016-03-02 19:08:33.867339418 +0100
@@ -46,7 +46,7 @@
 
 # Header file for CMake settings
 #hacky use layer include directory from loader repo
-set (LOADER_INCLUDE_DIR ${VULKANTOOLS_SOURCE_DIR}/../LoaderAndValidationLayers/include)
+set (LOADER_INCLUDE_DIR ${VULKANTOOLS_SOURCE_DIR}/../Vulkan-LoaderAndValidationLayers/include)
 include_directories(${PROJECT_SOURCE_DIR}/include ${LOADER_UNCLUDE_DIR})
 
 if(NOT WIN32)
@@ -109,7 +109,7 @@
     set(PYTHON_CMD "py")
 endif()
 
-set(LOADER_PY_DIR ${VULKANTOOLS_SOURCE_DIR}/../LoaderAndValidationLayers)
+set(LOADER_PY_DIR ${VULKANTOOLS_SOURCE_DIR}/../Vulkan-LoaderAndValidationLayers)
 
 # icd: Device dependent (DD) VULKAN components
 # tests: VULKAN tests
diff -dur -x '*~' vulkan-sdk-1.0.3.0.orig/VulkanTools/buildAndroid/android-generate.sh vulkan-sdk-1.0.3.0/VulkanTools/buildAndroid/android-generate.sh
--- vulkan-sdk-1.0.3.0.orig/VulkanTools/buildAndroid/android-generate.sh	2016-02-23 00:08:24.000000000 +0100
+++ vulkan-sdk-1.0.3.0/VulkanTools/buildAndroid/android-generate.sh	2016-03-02 19:07:43.720328878 +0100
@@ -19,8 +19,8 @@
 mkdir -p generated
 python ../vk-generate.py dispatch-table-ops layer > generated/vk_dispatch_table_helper.h
 
-python ../vk_helper.py --gen_enum_string_helper ../../LoaderAndValidationLayers/include/vulkan/vulkan.h --abs_out_dir generated
-python ../vk_helper.py --gen_struct_wrappers ../../LoaderAndValidationLayers/include/vulkan/vulkan.h --abs_out_dir generated
+python ../vk_helper.py --gen_enum_string_helper ../../Vulkan-LoaderAndValidationLayers/include/vulkan/vulkan.h --abs_out_dir generated
+python ../vk_helper.py --gen_struct_wrappers ../../Vulkan-LoaderAndValidationLayers/include/vulkan/vulkan.h --abs_out_dir generated
 
 python ../vk-layer-generate.py generic ../include/vulkan/vulkan.h > generated/generic_layer.cpp
 python ../vk-layer-generate.py api_dump ../include/vulkan/vulkan.h > generated/api_dump.cpp
diff -dur -x '*~' vulkan-sdk-1.0.3.0.orig/VulkanTools/icd/CMakeLists.txt vulkan-sdk-1.0.3.0/VulkanTools/icd/CMakeLists.txt
--- vulkan-sdk-1.0.3.0.orig/VulkanTools/icd/CMakeLists.txt	2016-02-23 00:08:24.000000000 +0100
+++ vulkan-sdk-1.0.3.0/VulkanTools/icd/CMakeLists.txt	2016-03-02 19:08:42.490731325 +0100
@@ -13,7 +13,7 @@
 endif()
   
 #hacky get include files from loader repo
-set (LOADER_INCLUDE_DIR ${VULKANTOOLS_SOURCE_DIR}/../LoaderAndValidationLayers/include)
+set (LOADER_INCLUDE_DIR ${VULKANTOOLS_SOURCE_DIR}/../Vulkan-LoaderAndValidationLayers/include)
 include_directories( ${LOADER_INCLUDE_DIR} )
 
 add_subdirectory(common)
diff -dur -x '*~' vulkan-sdk-1.0.3.0.orig/VulkanTools/layers/CMakeLists.txt vulkan-sdk-1.0.3.0/VulkanTools/layers/CMakeLists.txt
--- vulkan-sdk-1.0.3.0.orig/VulkanTools/layers/CMakeLists.txt	2016-03-02 19:06:25.066443989 +0100
+++ vulkan-sdk-1.0.3.0/VulkanTools/layers/CMakeLists.txt	2016-03-02 19:08:16.697222308 +0100
@@ -1,6 +1,6 @@
 cmake_minimum_required (VERSION 2.8.11)
 
-set (LOADER_INCLUDE_DIR ${VULKANTOOLS_SOURCE_DIR}/../LoaderAndValidationLayers/include)
+set (LOADER_INCLUDE_DIR ${VULKANTOOLS_SOURCE_DIR}/../Vulkan-LoaderAndValidationLayers/include)
 macro(run_vk_helper subcmd)
 	add_custom_command(OUTPUT ${ARGN}
 		COMMAND ${PYTHON_CMD} ${LOADER_PY_DIR}/vk_helper.py --${subcmd} ${LOADER_INCLUDE_DIR}/vulkan/vulkan.h --abs_out_dir ${CMAKE_CURRENT_BINARY_DIR}
@@ -72,8 +72,8 @@
 endif()
 
 #hacky use layer include directory from loader repo
-set (LOADER_LAYER_DIR ${VULKANTOOLS_SOURCE_DIR}/../LoaderAndValidationLayers/layers)
-set (LOADER_LOADER_DIR ${VULKANTOOLS_SOURCE_DIR}/../LoaderAndValidationLayers/loader)
+set (LOADER_LAYER_DIR ${VULKANTOOLS_SOURCE_DIR}/../Vulkan-LoaderAndValidationLayers/layers)
+set (LOADER_LOADER_DIR ${VULKANTOOLS_SOURCE_DIR}/../Vulkan-LoaderAndValidationLayers/loader)
 include_directories(
         ${LOADER_LAYER_DIR}
         ${LOADER_LOADER_DIR}
diff -dur -x '*~' vulkan-sdk-1.0.3.0.orig/VulkanTools/tests/CMakeLists.txt vulkan-sdk-1.0.3.0/VulkanTools/tests/CMakeLists.txt
--- vulkan-sdk-1.0.3.0.orig/VulkanTools/tests/CMakeLists.txt	2016-03-02 19:06:25.059777274 +0100
+++ vulkan-sdk-1.0.3.0/VulkanTools/tests/CMakeLists.txt	2016-03-02 19:08:28.020632945 +0100
@@ -88,7 +88,7 @@
 endif()
 
 #hacky get include files from loader repo
-set (LOADER_INCLUDE_DIR ${VULKANTOOLS_SOURCE_DIR}/../LoaderAndValidationLayers/include)
+set (LOADER_INCLUDE_DIR ${VULKANTOOLS_SOURCE_DIR}/../Vulkan-LoaderAndValidationLayers/include)
 include_directories(
      ${LOADER_INCLUDE_DIR}
     "${PROJECT_SOURCE_DIR}/tests/gtest-1.7.0/include"
@@ -131,13 +131,13 @@
 set (LIBVK "vulkan-${MAJOR}")
 if (${CMAKE_SYSTEM_NAME} MATCHES "Windows")
     set(LIBVK
-        ${VULKANTOOLS_SOURCE_DIR}/../LoaderAndValidationLayers/${BUILDTGT_DIR}/loader/${CMAKE_CFG_INTDIR}/vulkan-${MAJOR}.lib
+        ${VULKANTOOLS_SOURCE_DIR}/../Vulkan-LoaderAndValidationLayers/${BUILDTGT_DIR}/loader/${CMAKE_CFG_INTDIR}/vulkan-${MAJOR}.lib
     )
 endif()
 
 if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
     set(LIBVK
-        ${VULKANTOOLS_SOURCE_DIR}/../LoaderAndValidationLayers/build/loader/${CMAKE_CFG_INTDIR}/libvulkan.so
+        ${VULKANTOOLS_SOURCE_DIR}/../Vulkan-LoaderAndValidationLayers/build/loader/${CMAKE_CFG_INTDIR}/libvulkan.so
     )
 endif()
 add_executable(vkbase init.cpp ${COMMON_CPP})
diff -dur -x '*~' vulkan-sdk-1.0.3.0.orig/VulkanTools/vk-generate.py vulkan-sdk-1.0.3.0/VulkanTools/vk-generate.py
--- vulkan-sdk-1.0.3.0.orig/VulkanTools/vk-generate.py	2016-02-23 00:08:24.000000000 +0100
+++ vulkan-sdk-1.0.3.0/VulkanTools/vk-generate.py	2016-03-02 19:08:50.470785391 +0100
@@ -27,7 +27,7 @@
 
 import sys
 import imp
-vulkan = imp.load_source('vulkan', '../../../../LoaderAndValidationLayers/vulkan.py')
+vulkan = imp.load_source('vulkan', '../../../../Vulkan-LoaderAndValidationLayers/vulkan.py')
 
 
 def generate_get_proc_addr_check(name):
diff -dur -x '*~' vulkan-sdk-1.0.3.0.orig/VulkanTools/vk-layer-generate.py vulkan-sdk-1.0.3.0/VulkanTools/vk-layer-generate.py
--- vulkan-sdk-1.0.3.0.orig/VulkanTools/vk-layer-generate.py	2016-02-23 00:08:24.000000000 +0100
+++ vulkan-sdk-1.0.3.0/VulkanTools/vk-layer-generate.py	2016-03-02 19:09:51.384527220 +0100
@@ -37,8 +37,8 @@
 import re
 
 import imp
-vulkan = imp.load_source('vulkan', '../../../LoaderAndValidationLayers/vulkan.py')
-vk_helper = imp.load_source('vulkan', '../../../LoaderAndValidationLayers/vk_helper.py')
+vulkan = imp.load_source('vulkan', '../../../Vulkan-LoaderAndValidationLayers/vulkan.py')
+vk_helper = imp.load_source('vulkan', '../../../Vulkan-LoaderAndValidationLayers/vk_helper.py')
 from source_line_info import sourcelineinfo
 from collections import defaultdict
 
diff -dur -x '*~' vulkan-sdk-1.0.3.0.orig/VulkanTools/vktrace/CMakeLists.txt vulkan-sdk-1.0.3.0/VulkanTools/vktrace/CMakeLists.txt
--- vulkan-sdk-1.0.3.0.orig/VulkanTools/vktrace/CMakeLists.txt	2016-02-23 00:08:24.000000000 +0100
+++ vulkan-sdk-1.0.3.0/VulkanTools/vktrace/CMakeLists.txt	2016-03-02 19:09:41.847797072 +0100
@@ -26,7 +26,7 @@
 endif()
 
 #hacky get include files from loader repo
-set (VKTRACE_VULKAN_INCLUDE_DIR ${VULKANTOOLS_SOURCE_DIR}/../LoaderAndValidationLayers/include)
+set (VKTRACE_VULKAN_INCLUDE_DIR ${VULKANTOOLS_SOURCE_DIR}/../Vulkan-LoaderAndValidationLayers/include)
 include_directories( ${VKTRACE_VULKAN_INCLUDE_DIR} )
 
 message("")
diff -dur -x '*~' vulkan-sdk-1.0.3.0.orig/VulkanTools/vktrace/src/vktrace_extensions/vktracevulkan/CMakeLists.txt vulkan-sdk-1.0.3.0/VulkanTools/vktrace/src/vktrace_extensions/vktracevulkan/CMakeLists.txt
--- vulkan-sdk-1.0.3.0.orig/VulkanTools/vktrace/src/vktrace_extensions/vktracevulkan/CMakeLists.txt	2016-02-23 00:08:24.000000000 +0100
+++ vulkan-sdk-1.0.3.0/VulkanTools/vktrace/src/vktrace_extensions/vktracevulkan/CMakeLists.txt	2016-03-02 19:09:32.241066284 +0100
@@ -30,14 +30,14 @@
 
 if (${CMAKE_SYSTEM_NAME} MATCHES "Windows")
     set(VKTRACE_VULKAN_LIB
-        ${VULKANTOOLS_SOURCE_DIR}/../LoaderAndValidationLayers/${BUILDTGT_DIR}/loader/${CMAKE_CFG_INTDIR}/vulkan-${MAJOR}.lib
+        ${VULKANTOOLS_SOURCE_DIR}/../Vulkan-LoaderAndValidationLayers/${BUILDTGT_DIR}/loader/${CMAKE_CFG_INTDIR}/vulkan-${MAJOR}.lib
     )
 endif()
 
  
 if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
     set(VKTRACE_VULKAN_LIB
-        ${VULKANTOOLS_SOURCE_DIR}/../LoaderAndValidationLayers/build/loader/libvulkan.so
+        ${VULKANTOOLS_SOURCE_DIR}/../Vulkan-LoaderAndValidationLayers/build/loader/libvulkan.so
      )
 endif()
 
diff -dur -x '*~' vulkan-sdk-1.0.3.0.orig/VulkanTools/vktrace/vktrace_generate.py vulkan-sdk-1.0.3.0/VulkanTools/vktrace/vktrace_generate.py
--- vulkan-sdk-1.0.3.0.orig/VulkanTools/vktrace/vktrace_generate.py	2016-02-23 00:08:24.000000000 +0100
+++ vulkan-sdk-1.0.3.0/VulkanTools/vktrace/vktrace_generate.py	2016-03-02 19:09:20.920990500 +0100
@@ -37,7 +37,7 @@
 sys.path.append(main_path)
 from source_line_info import sourcelineinfo
 import imp
-vulkan = imp.load_source('vulkan', '%s/../LoaderAndValidationLayers/vulkan.py' % main_path)
+vulkan = imp.load_source('vulkan', '%s/../Vulkan-LoaderAndValidationLayers/vulkan.py' % main_path)
 
 # vulkan.py doesn't include all the extensions (debug_report missing)
 headers = []
