diff -dur vulkan-sdk-1.0.3.0.orig/LunarGLASS/CMakeLists.txt vulkan-sdk-1.0.3.0/LunarGLASS/CMakeLists.txt
--- vulkan-sdk-1.0.3.0.orig/LunarGLASS/CMakeLists.txt	2016-02-20 01:27:02.000000000 +0100
+++ vulkan-sdk-1.0.3.0/LunarGLASS/CMakeLists.txt	2016-02-24 13:49:27.000000000 +0100
@@ -49,7 +49,7 @@
     # Get system library dependencies from llvm-config.
     execute_process(COMMAND ${LLVM_BUILD}/install/usr/local/bin/llvm-config --ldflags OUTPUT_VARIABLE LLVM_LD_FLAGS)
     # Convert to a list of link flags.
-    string(REPLACE " " ";" LLVM_LD_FLAGS_LIST ${LLVM_LD_FLAGS})
+    string(REPLACE " " ";" LLVM_LD_FLAGS_LIST "${LLVM_LD_FLAGS}")
     foreach(LIB ${LLVM_LD_FLAGS_LIST})
       if (${LIB} MATCHES "^-l")
         # Get the library's name.
