diff -dur -x '*.orig' VulkanTools-sdk-1.0.32.0.orig/layersvt/CMakeLists.txt VulkanTools-sdk-1.0.32.0/layersvt/CMakeLists.txt
--- VulkanTools-sdk-1.0.32.0.orig/layersvt/CMakeLists.txt	2016-11-10 18:52:27.000000000 +0100
+++ VulkanTools-sdk-1.0.32.0/layersvt/CMakeLists.txt	2016-11-23 14:18:09.000000000 +0100
@@ -30,7 +30,7 @@
     VkLayer_device_simulation
     )
 
-set(VK_LAYER_RPATH /usr/lib/x86_64-linux-gnu/vulkan/layer:/usr/lib/i386-linux-gnu/vulkan/layer)
+set(VK_LAYER_RPATH "${CMAKE_INSTALL_LIBDIR}/vulkan/layer")
 set(CMAKE_INSTALL_RPATH ${VK_LAYER_RPATH})
 
 if (WIN32)
