diff -dur -x '*~' -x '*.orig' vulkan-sdk-1.0.3.0.orig/Vulkan-LoaderAndValidationLayers/layers/CMakeLists.txt vulkan-sdk-1.0.3.0/Vulkan-LoaderAndValidationLayers/layers/CMakeLists.txt
--- vulkan-sdk-1.0.3.0.orig/Vulkan-LoaderAndValidationLayers/layers/CMakeLists.txt	2016-02-24 10:16:45.000000000 +0100
+++ vulkan-sdk-1.0.3.0/Vulkan-LoaderAndValidationLayers/layers/CMakeLists.txt	2016-02-24 10:17:24.000000000 +0100
@@ -26,7 +26,7 @@
     VkLayer_device_limits
     )
 
-set(VK_LAYER_RPATH /usr/lib/x86_64-linux-gnu/vulkan/layer:/usr/lib/i386-linux-gnu/vulkan/layer)
+set(VK_LAYER_RPATH "${CMAKE_INSTALL_LIBDIR}/vulkan/layer")
 set(CMAKE_INSTALL_RPATH ${VK_LAYER_RPATH})
 
 if (NOT WIN32)
diff -dur -x '*~' -x '*.orig' vulkan-sdk-1.0.3.0.orig/VulkanTools/layers/CMakeLists.txt vulkan-sdk-1.0.3.0/VulkanTools/layers/CMakeLists.txt
--- vulkan-sdk-1.0.3.0.orig/VulkanTools/layers/CMakeLists.txt	2016-02-23 00:08:24.000000000 +0100
+++ vulkan-sdk-1.0.3.0/VulkanTools/layers/CMakeLists.txt	2016-02-24 10:17:37.000000000 +0100
@@ -23,7 +23,7 @@
     VkLayer_screenshot
     )
 
-set(VK_LAYER_RPATH /usr/lib/x86_64-linux-gnu/vulkan/layer:/usr/lib/i386-linux-gnu/vulkan/layer)
+set(VK_LAYER_RPATH "${CMAKE_INSTALL_LIBDIR}/vulkan/layer")
 set(CMAKE_INSTALL_RPATH ${VK_LAYER_RPATH})
 
 if (NOT WIN32)
