--- VulkanTools-sdk-1.0.39.1/layersvt/CMakeLists.txt.orig	2017-05-27 08:25:39.348712996 +0200
+++ VulkanTools-sdk-1.0.39.1/layersvt/CMakeLists.txt	2017-05-27 08:39:25.502036896 +0200
@@ -105,7 +105,7 @@
     target_link_Libraries(VkLayer_${target} VkLayer_utilsvt)
     add_dependencies(VkLayer_${target} generate_vt_helpers generate_api_cpp generate_api_h)
     set_target_properties(VkLayer_${target} PROPERTIES LINK_FLAGS "-Wl,-Bsymbolic")
-    install(TARGETS VkLayer_${target} DESTINATION ${PROJECT_BINARY_DIR}/install_staging)
+    install(TARGETS VkLayer_${target} DESTINATION ${CMAKE_INSTALL_LIBDIR})
     endmacro()
 endif()
 
@@ -166,7 +166,7 @@
     add_library(VkLayer_utilsvt STATIC ../layers/vk_layer_config.cpp ../layers/vk_layer_extension_utils.cpp ../layers/vk_layer_utils.cpp)
 else()
     add_library(VkLayer_utilsvt SHARED ../layers/vk_layer_config.cpp ../layers/vk_layer_extension_utils.cpp ../layers/vk_layer_utils.cpp)
-    install(TARGETS VkLayer_utilsvt DESTINATION ${PROJECT_BINARY_DIR}/install_staging)
+    install(TARGETS VkLayer_utilsvt DESTINATION ${CMAKE_INSTALL_LIBDIR})
 endif()
 
 # VulkanTools layers
