diff -dur -x '*~' VulkanTools-sdk-1.0.32.0.orig/layers/threading.h VulkanTools-sdk-1.0.32.0/layers/threading.h
--- VulkanTools-sdk-1.0.32.0.orig/layers/threading.h	2016-11-10 18:52:27.000000000 +0100
+++ VulkanTools-sdk-1.0.32.0/layers/threading.h	2016-11-23 15:54:32.000000000 +0100
@@ -26,7 +26,7 @@
 #include "vk_layer_config.h"
 #include "vk_layer_logging.h"
 
-#if defined(__LP64__) || defined(_WIN64) || defined(__x86_64__) || defined(_M_X64) || defined(__ia64) || defined(_M_IA64) ||       \
+#if defined(__LP64__) || defined(_WIN64) || (defined(__x86_64__) && !defined(__ILP32__)) || defined(_M_X64) || defined(__ia64) || defined(_M_IA64) ||       \
     defined(__aarch64__) || defined(__powerpc64__)
 // If pointers are 64-bit, then there can be separate counters for each
 // NONDISPATCHABLE_HANDLE type.  Otherwise they are all typedef uint64_t.
